README.md

```diff
@@ -68,6 +68,15 @@ we can only guarantee compatibility with the TensorFlow versions which it was te
 Warnings will be emitted when importing `tensorflow_addons` if your TensorFlow version does not match 
 what it was tested against.
 
+To ensure you install a compatible version of `tensorflow_addons` for your existing version of
+TensorFlow, you can install with the `tensorflow` extra requirement:
+
+```
+pip install tensorflow-addons[tensorflow]
+```
```

Comment:
Pending the answer to https://github.com/tensorflow/addons/issues/2111#issuecomment-679510888 I think we could even move this up to the primary "Install TFA" block

Reply:
As discussed in that issue, reworded this section to clarify that the effect will be to ensure a compatible version of TensorFlow is installed, not the reverse (a compatible TFA for your TF install).  Also moved to the `Installation` section as suggested.  Let me know what you think!